locale/ru.po

```diff
@@ -659,7 +659,7 @@ msgstr "pikaur требует Python >= 3.7 для запуска."
 #: pikaur/main.py:96
 #, fuzzy
```

Comment:
please remove `#, fuzzy` since you added the correct translation (or run `make`, i think it should remove it automatically)